eve-tools/bpftrace-compiler/edgeview.go

```diff
@@ -0,0 +1,255 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"io/fs"
+	"log"
+	"net"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"syscall"
+	"time"
+	"unicode"
+
+	"github.com/creack/pty"
+)
+
+type edgeviewRun struct {
+	cmd    *exec.Cmd
+	ctx    context.Context
+	cancel func()
+	path   string
+	port   uint16
+	tty    *os.File
+}
+
+func pathToScript(path string) string {
+	scripts, err := filepath.Glob(path)
+	if err != nil {
+		log.Fatalf("could not glob '%s': %v", path, err)
+	}
+
+	newestScript := ""
+	newestTime := time.Unix(0, 0)
+	for _, script := range scripts {
+		st, err := os.Stat(script)
+		if err != nil {
+			continue
+		}
+
+		if st.ModTime().After(newestTime) {
+			newestTime = st.ModTime()
+			newestScript = script
+		}
+	}
+
+	return newestScript
+}
+
+func (er edgeviewRun) enablePprof() error {
+	edgeviewParam := "pprof/on"
+	matchOutput := "=== System: <pprof/on> ==="
```

Comment:
Can we mention where this pattern comes from in the comment? That way, we'll know where to get it if it changes later.

Reply:
Yes, that's a good point. I added it.